app/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router';
+import { hashHistory } from 'react-router';
```

Comment:
https://github.com/reactjs/react-router/blob/master/docs/guides/Histories.md#creatememoryhistory isn't this one better, because you don't heave a real history in the electron environment, if i am not mistaken?